lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dva/DvaThirdPartyDocumentGateway.java

```diff
@@ -275,9 +275,25 @@ private DocumentCheckResult responseHandler(
     }
 
     private HttpPost requestBuilder(
-            URI endpointUri, String userName, String password, String requestBody)
+            URI endpointUri,
+            String userName,
+            String password,
+            String requestBody,
+            DvaPayload dvaPayload)
             throws UnsupportedEncodingException {
         HttpPost request = new HttpPost(endpointUri);
+        if (configurationService.isDvaPerformanceStub()) {
```

Comment:
I think this can live on 127 it makes it clearer thats not really part of the request more an addition